base_layer/tari_mining_helper_ffi/src/lib.rs

```diff
@@ -233,6 +245,153 @@ pub unsafe extern "C" fn inject_nonce(header: *mut ByteVector, nonce: c_ulonglon
     (*header).0 = buffer;
 }
 
+/// Injects a coinbase into a blocktemplate
+///
+/// ## Arguments
+/// `block_template_bytes` - The block template as bytes, serialized with borsh.io
+/// `value` - The value of the coinbase
+/// `stealth_payment` - Boolean value, is this a stealh payment or normal one-sided
+/// `revealed_value_proof` - Boolean value, should this use the reveal value proof, or BP+
+/// `wallet_payment_address` - The address to pay the coinbase to
+/// `coinbase_extra` - The value of the coinbase extra field
+/// `network` - The value of the network
+///
+/// ## Returns
+/// `block_template_bytes` - The updated block template
+/// `error_out` - Error code returned, 0 means no error
+///
+/// # Safety
+/// None
+#[allow(clippy::too_many_lines)]
+#[no_mangle]
+pub unsafe extern "C" fn inject_coinbase(
+    block_template_bytes: *mut ByteVector,
+    value: c_ulonglong,
```

Comment:
Maybe `reward` is better?
```suggestion
    reward: c_ulonglong,
```

Reply:
No specifically moved to value as to get away from fee and reward